kura/org.eclipse.kura.request.handler.jaxrs/META-INF/MANIFEST.MF

```diff
@@ -10,6 +10,7 @@ Bundle-ActivationPolicy: lazy
 Import-Package: com.google.gson,
  javax.ws.rs;version="2.0.1",
  javax.ws.rs.core;version="2.0.1",
+ org.apache.commons.io;version="2.4.0",
```

Comment:
could we use a version range ?

Reply:
Usually we use version ranges only for packages exported by Kura, because we are sure that they use semantic versioning.